test/k6/src/tests/orders-org-no.js

```
@@ -33,26 +33,22 @@ import { orgNosYt01 } from "../data/orgnos.js";
-const resourceId = __ENV.resourceId;
-const scopes = "altinn:serviceowner/notifications.create";
-
+import { scopes, resourceId, environment, yt01Environment, options } from "../shared/variables.js";
 
 const emailOrderRequestJson = JSON.parse(
     open("../data/orders/01-email-request.json")
 );
 
-const labels = [post_mail_order, get_mail_notifications, post_sms_order, get_sms_notifications];
-
 export const options = {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'options'. Consider to delete it or rename it.

'options' is defined here:


(lint/suspicious/noRedeclare)